docs/api/ecs/query.md

```diff
@@ -19,13 +19,13 @@ import { Position, Velocity, Acceleration } from './components';
 
 const world = new World({...});
 const query = world.createQuery(Position, ANY(Velocity, NOT(Acceleration)));
-query.forEach(entityId => {
+query.items().forEach((entityId) => {
```

Comment:
We should swap the order of these; `const entityId of query.iter()` should come first as I know the syntax is not desired, but it will (for now) be the way we should be doing it for most cases.